src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityEnergyHatch.java

```diff
@@ -154,4 +166,37 @@ public void getSubItems(CreativeTabs creativeTab, NonNullList<ItemStack> subItem
             }
         }
     }
+
+    public void checkTerrainExplosion() {
```

Comment:
This method could be put in GTUtility or in MTE, so you don't have to copy and paste it for both the tiered MTEs and the energy hatches.